packages/langfuse-wrapper/src/langfuse_wrapper/langfuse_wrapper.py

```diff
@@ -0,0 +1,96 @@
+"""
+Langfuse Integration Module
+
+This module provides integration with Langfuse, a tool for monitoring and tracing LLM applications.
+
+Usage:
+    Import this module to enable Langfuse integration.
+    It automatically checks for Langfuse credentials in the .env.langfuse file and for a running Langfuse server.
+    If these are found, it will set up the necessary client and context for tracing.
+
+Note:
+    Run setup_langfuse.sh which automates the steps for running local Langfuse.
+"""
+
+import os
+import logging
+from typing import Callable
+from dotenv import load_dotenv
+from langfuse.decorators import langfuse_context
+import sys
+from io import StringIO
+from pathlib import Path
+from functools import wraps  # Add this import
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
this will change the logging behavior of any main processes that import this file, which we want to avoid. I'd leave the log level and formatting configuration to the main process